README.md

```diff
@@ -0,0 +1,31 @@
+gonat [![Build Status](https://drone.lantern.io/api/badges/getlantern/gonat/status.svg)](https://drone.lantern.io/getlantern/gonat)&nbsp;[![GoDoc](https://godoc.org/github.com/getlantern/gonat?status.png)](http://godoc.org/github.com/getlantern/gonat)
+==========
+
+This library only works on Linux.
+
+Dependencies are managed using Go modules. If using a version of Go prior to 1.13, use the environment
+variable `GO111MODULE=on` to enable use of modules.
+
+In order to work, this library needs to be able to open raw sockets and update the conntrack table
+via netlink. You can give the binary the correct capabilities with:
+
+`sudo setcap CAP_NET_RAW,CAP_NET_ADMIN+ep`
```

Comment:
Something like `sudo setcap CAP_NET_RAW,CAP_NET_ADMIN+ep <binary>` might be helpful for dummies like me who skim and just try to run the commands =P

Reply:
Good idea, fixing.